refactor(curriculum): CSS Animations Quiz

estefaniacn opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or Gitpod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #58494

<!-- Feel free to add any additional description of changes below this line -->

estefaniacn committed:
refactor(curriculum): CSS animations quiz

moT01 reviewed (changes requested):
looking good @estefaniacn 👍 Found one issue...


Question 7: I don't think the correct answer is clear.
"They may not work correctly on older browsers." - We don't mention it, but it's technically true. Very old browsers don't support it.
"They may distract users from the main content of the website." - Our lecture says they "may be distracting"
"They may decrease website loading speed." - Our lecture says they "can lead to poor performance"
"They may distract users from the main content of the website." (Correct answer) - Our lecture says they "may be problematic for users with certain accessibility needs"

So it seems like any of the answers could be correct. We should change this one up I think.

estefaniacn committed:
fix(curriculum): apply code review suggestions

estefaniacn commented:
Thank you for your feedback, @moT01. I rewrote the question to ask about the option that is not a potential concern with CSS animations and rewrote the options a little bit. Please let me know what you think. 👍

moT01 commented:
I think it's still a little confusing. Like I said, we don't mention that "They may not work correctly on older browsers." So users won't know that disctractor for sure.

I also mentioned `"They may decrease website loading speed." - Our lecture says they "can lead to poor performance"` - I think I made a mistake here and thought that "decrease" would mean slower, but really it's faster I think? I think we shouldn't use "increase" or "decrease".

How about this:

```
Which of the following is NOT a potential concern with CSS animations?

--distractors--

They may cause discomfort or physical harm to certain users.

Users may find them distracting.

Overuse can lead to poor performance.

--answer--

They can enhance user experience.
```

estefaniacn committed:
fix(curriculum): apply code review suggestions

estefaniacn commented:
I like how you rephrased the distractor and the correct answer. I implemented these changes. 👍
